packages/grafana-i18n/src/i18n.tsx

```diff
@@ -189,10 +193,12 @@ export const t: TFunction = (id: string, defaultMessage: string, values?: Record
 };
 
 export function useTranslate() {
+  initDefaultI18nInstance();
```

Comment:
technically not needed in both `useTranslate` and `t`. Normally I would say leave it in just `useTranslate`, but if we're getting rid of it, then it would need to stay in `t`.

This isn't really doing harm here - If @hugohaggmark is going to remove `useTranslate` shortly, maybe can just leave this PR as is to avoid the extra push.

Reply:
Opened https://github.com/grafana/grafana/pull/106556